### DISCUSSION TITLE
Visually group dates on the same day

### DISCUSSION THREAD

**Discussion by dansan** [lukevella/rallly#313] [Ideas]
All dates are listed from left to right without grouping. It is not easy to see how dates belong together / are repeated.

If you look at https://dud-poll.inf.tu-dresden.de/coffeebreak/ it is very easy to see a pattern:

![image](https://user-images.githubusercontent.com/709115/189297161-925a3d71-30d1-41e0-97db-9514177d13bf.png)

It is a common use case to have the same 2-3 dates on multiple days to choose from. The grouping makes it fast for the end user to select for example dates after 17°° on multiple days.

**Comment by lukevella** [comment]
Thanks @dansan, I like this idea a lot. I tried playing with something similar in the past but found that there are usability issues when there are a large number of options which is not uncommon. You might end up with the date or month being offscreen which isn't great. I haven't given up on this though and will continue to try and find a good solution.
